src/theory/strings/theory_strings_rewriter.cpp

```diff
@@ -739,10 +779,11 @@ Node TheoryStringsRewriter::rewriteMembership(TNode node) {
     CVC4::String s = x.getConst<String>();
     retNode = NodeManager::currentNM()->mkConst( testConstStringInRegExp( s, 0, r ) );
   } else if(r.getKind() == kind::REGEXP_SIGMA) {
-    Node one = NodeManager::currentNM()->mkConst( ::CVC4::Rational(1) );
+    Node one = NodeManager::currentNM()->mkConst(Rational(1));
```

Comment:
Nit: `NodeManager::currentNM()` -> `nm` (since you are already changing this line)